### ISSUE TITLE
 mermaid gantt graph in small screen

### ISSUE THREAD

**Issue by bosconb** [matcornic/hugo-theme-learn#257]
when theme embed a mermaid gantt graph, the abscissa scale shows a terrible result in mobile (iphone 6/7 plus screen),any idea?

![ mermaid gantt graph in small screen](https://user-images.githubusercontent.com/47100031/55157216-d90d9100-5196-11e9-8a80-50a3fa62c9d2.jpeg)

**Comment by matalo33** [comment]
Agreed, it does look quite bad! Is this an issue with mermaid or the theme implementation of mermaid?

**Comment by McShelby** [comment]
Mermaid generates a `width=100%` for certain diagram types - e.g the gantt diagram. On a narrow website this may cause the observed distortion.

The mermaid docs expose this behaviour, too, as seen [here](https://mermaid-js.github.io/mermaid/diagrams-and-syntax-and-examples/gantt.html).

Also related to this issue: If mermaid does not generate a `width=100%` the diagramm may overflow out of view if the browser window is too narrow - *without* generating scroll bars. This is clearly an issue of our CSS and also affects print output.

Nevertheless I don't have an idea how to fix this in a satisfying way for all users.
